iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/RemoteReactivePowerControlImpl.java

```diff
@@ -6,40 +6,104 @@
  */
 package com.powsybl.iidm.network.impl.extensions;
 
-import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.commons.util.trove.TBooleanArrayList;
 import com.powsybl.iidm.network.Generator;
 import com.powsybl.iidm.network.Terminal;
 import com.powsybl.iidm.network.extensions.RemoteReactivePowerControl;
+import com.powsybl.iidm.network.impl.AbstractMultiVariantIdentifiableExtension;
+import gnu.trove.list.array.TDoubleArrayList;
+
+import java.util.ArrayList;
 
 /**
  * @author Bertrand Rix <bertrand.rix at artelys.com>
  */
-public class RemoteReactivePowerControlImpl extends AbstractExtension<Generator> implements RemoteReactivePowerControl {
+public class RemoteReactivePowerControlImpl extends AbstractMultiVariantIdentifiableExtension<Generator> implements RemoteReactivePowerControl {
 
-    double targetQ;
+    private TDoubleArrayList targetQ;
 
-    Terminal regulatingTerminal;
+    private final ArrayList<Terminal> regulatingTerminal;
```

Comment:
Might not be linked directly to this PR but should regulatingTerminal be multi-variant in regulating equipment? Because they are not for now @annetill @sylvlecl

Reply:
Ah you're right we should remain consistent with the base model, I thought it was multi-variant.
Actually now I think that Anne mentioned yesterday that in CGMES, that info is in the EQ file, so it makes sense to keep it this way.

@CBiasuzzi : could you revert just this one please?